src/components/ChannelList.vue

```diff
@@ -155,6 +155,9 @@ export default {
         joinChannel(channelName) {
             this.$state.addBuffer(this.network.id, channelName);
             this.network.ircClient.join(channelName);
+            if (this.$state.ui.app_width <= 769) {
```

Comment:
I've used this in more than one place, not sure if it should be made into a method in state or somewhere more appropriate